app/src/main/res/layout/change_password.xml

```diff
@@ -7,43 +7,32 @@
     android:orientation="vertical"
     tools:context=".ui.user.ChangePasswordFragment">
 
-    <TextView
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:text="@string/change_password"
-        android:textFontWeight="600"
-        android:textColor="@color/black"
-        android:textSize="20dp"
-        android:textStyle="normal"
-        android:layout_marginTop="50dp"
-        android:layout_marginBottom="20dp"
-        android:layout_gravity="center_horizontal"></TextView>
-
     <TextView
         android:id="@+id/text_old_password"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_marginStart="20dp"
+        android:layout_marginStart="20sp"
         android:clickable="true"
         android:fontFamily="@font/font_poppins"
         android:text="@string/old_password"
         android:textColor="@color/black"
-        android:textSize="19dp"
+        android:textSize="19sp"
         android:textStyle="normal"
-        tools:ignore="TouchTargetSizeCheck"></TextView>
+        tools:ignore="TouchTargetSizeCheck"/>
 
     <com.google.android.material.textfield.TextInputLayout
         android:id="@+id/TextOldPassword"
         style="@style/Widget.MaterialComponents.TextInputLayout.OutlinedBox"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:layout_marginStart="20dp"
-        android:layout_marginEnd="20dp"
-        android:layout_marginBottom="20dp"
-        app:boxCornerRadiusBottomEnd="10dp"
-        app:boxCornerRadiusBottomStart="10dp"
-        app:boxCornerRadiusTopEnd="10dp"
-        app:boxCornerRadiusTopStart="10dp"
+        android:layout_marginStart="20sp"
```

Comment:
sp units should only be for text size: see android docs
_When defining text sizes, you can instead use scalable pixels (sp) as your units. The sp unit is the same size as a dp, by default, but it resizes based on the user's preferred text size. Never use sp for layout sizes._

Reply:
addressed